_layouts/home.html

```diff
@@ -1,78 +1,93 @@
 ---
 layout: default
 ---
+<div id="home">
+{% include header.html %}
 
+<div class="header-image">
+  <div class="header-image-caption">
+    art made with mypaint by <a href="http://nrekkvan.deviantart.com">nrekkvan</a>
```

Comment:
Probably want to add it to the Front Matter to make interchanging the attrb. info. much easier.